include/crm/pengine/internal.h

```diff
@@ -548,9 +548,17 @@ bool pe__bundle_needs_remote_name(pe_resource_t *rsc,
 const char *pe__add_bundle_remote_name(pe_resource_t *rsc,
                                        pe_working_set_t *data_set,
                                        xmlNode *xml, const char *field);
+
+enum pcmk__location_e {
+    pcmk__loc_allocated = (1 << 0),
+    pcmk__loc_running = (1 << 1),
+    /* pcmk__loc_pending = (1 << 2), */
```

Comment:
Nothing wrong with adding an unused value now.

I'm thinking "current" instead of "running" for consistency with e.g. pe__current_node(), plus I believe a pending start counts as an active node

Rename the enum to avoid confusion with location constraints (stick resource or rsc in there, and/or use node instead of location)

Put the new enum and argument in a refactor commit so the actual fix is more obvious

Reply:
Done